cmd/mario/main.go

```diff
@@ -113,112 +66,156 @@ func main() {
 					return err
 				}
 				for _, i := range indexes {
-					fmt.Printf(`
-Name: %s
-  Documents: %d
-  Health: %s
-  Status: %s
-  UUID: %s
-  Size: %s
-`, i.Index, i.DocsCount, i.Health, i.Status, i.UUID, i.StoreSize)
+					fmt.Printf("Name: %s\n\tDocuments: %d\n\tHealth: %s\n\tStatus: %s\n\tUUID: %s\n\tSize: %s\n\n", i.Index, i.DocsCount, i.Health, i.Status, i.UUID, i.StoreSize)
 				}
 				return nil
 			},
-		},
+    },
 		{
-			Name:  "aliases",
-			Usage: "List Elasticsearch aliases and associated indexes",
+			Name:     "ping",
+			Usage:    "Ping Elasticsearch",
+      Category: "Elasticsearch actions",
 			Action: func(c *cli.Context) error {
 				es, err := client.NewESClient(url, v4)
 				if err != nil {
 					return err
 				}
-				aliases, err := es.Aliases()
+				res, err := es.Ping(url)
 				if err != nil {
 					return err
 				}
-				for _, a := range aliases {
-					fmt.Printf(`
-Alias: %s
-  Index: %s
-`, a.Alias, a.Index)
-				}
+				fmt.Printf("Name: %s\nCluster: %s\nVersion: %s\nLucene version: %s", res.Name, res.ClusterName, res.Version.Number, res.Version.LuceneVersion)
 				return nil
 			},
 		},
+    // Index-specific commands
 		{
-			Name:  "ping",
-			Usage: "Ping Elasticsearch",
+			Name:      "ingest",
+			Usage:     "Parse and ingest the input file. By default, ingests into the current production index for the provided source.",
+			ArgsUsage: "[filepath, use format 's3://bucketname/objectname' for s3]",
+      Category:  "Index actions",
+			Flags:     []cli.Flag{
+				&cli.StringFlag{
+					Name:  	  "source",
+					Aliases:  []string{"s"},
+					Usage: 	  "Source system of metadata file to process. Must be one of [aleph, aspace, dspace, mario]",
+					Required: true,
+				},
+				&cli.StringFlag{
+					Name:  	 "consumer",
+					Aliases: []string{"c"},
+					Value: 	 "es",
+					Usage: 	 "Consumer to use. Must be one of [es, json, title, silent]",
+				},
+        &cli.BoolFlag{
+          Name: "new",
```

Comment:
I like this new functionality _a lot_. For consistency though I think aliasing as `-n` might be good?

Reply:
Hmm. We don't do that for the other boolean flags?

Reply:
Yeah that's fair. I guess I wasn't looking at it as "booleans don't have shortcuts" and more like "why do some things have shortcuts and some don't". I'd like us to be consistent, and I'll accept your argument that this is consistent with this type of flag.